### ISSUE TITLE
Python: 'GPULoggingThread' object and 'CPULoggingThread' has no attribute 'isAlive'

### ISSUE THREAD

**Issue by mwup** [comet-ml/issue-tracking#455]
## Describe the Bug
Hello :)

I recently got the following error:
'GPULoggingThread' object has no attribute 'isAlive'
and
'CPULoggingThread' object has no attribute 'isAlive'.
I read on another thread: https://github.com/jupyter-vim/jupyter-vim/issues/51, that "isAlive()" is deprecated and "is_alive()" is being used.

I changed the relevant lines (458 and 473) from the comet_ml/experiment.py file for it to work again:
l.458: self.gpu_thread.isAlive() -> self.gpu_thread.is_alive()
l.473: self.cpu_thread.isAlive() -> self.cpu_thread.is_alive()

## Where is the issue?
- [x] Comet Python SDK

## To Reproduce
I downloaded comet ml for python from: https://anaconda.org/comet_ml/comet_ml using conda install -c comet_ml comet_ml.
Then I trained a pytorch model on the gpu and logged a metric and image.

experiment.log_image(img.detach().cpu(), ...)

## Screenshots or GIFs
![image](https://user-images.githubusercontent.com/75084005/148950018-696a482d-147d-432d-8c84-3cfce1b56f48.png)
![image](https://user-images.githubusercontent.com/75084005/148950059-21b1c0a6-325f-48b2-82b3-19dc1e4d29f9.png)

**Comment by DN6** [comment]
Hello @mwup. Thanks for flagging this. Looks like a change from Python 3.8 to 3.9 might be causing the issue. I will bring this to our engineering team's attention.

**Comment by DN6** [comment]
Would it be possible to run your experiment using Python 3.8? That should solve your issue until we can formally resolve this.

**Comment by Lothiraldan** [comment]
Hello @mwup, the Comet Python SDK is fully compatible with Python 3.9. Could you check which version you are using? You can find this information with the following command: `conda list | grep comet`. You can also try to update your comet_ml version.

Let us know if you have any follow-up question.

**Comment by shirelattia19** [comment]
Hi I have the same problem already , what should i do ?

**Comment by Lothiraldan** [comment]
Hi @shirelattia19, which Python version and comet version are you using?

**Comment by shirelattia19** [comment]
Hi @Lothiraldan 
python                    3.9.7                h12debd9_1  
comet_ml                  3.1.6                      py_0    comet_ml
